### ISSUE TITLE
Size estimation for customer how not move data to M365 yet

### ISSUE THREAD

**Issue by marcohorstmann** [VeeamHub/veeam-calculators#6]
**Is your feature request related to a problem? Please describe.**
Customers more often start to inform themself before starting the migration to M365. It would be nice to figure out
at this stage how many space they need to move onprem Exchange or Sharepoint to M365
**Describe the solution you'd like**
A way to get needed data from Exchange (must) and Sharepoint (nice to have because Sharepoint onPrem is rare)

**Comment by shapedthought** [comment]
Certainly possible, looking at the Exchange PowerShell module all the parts are there:

https://learn.microsoft.com/en-us/powershell/module/exchange/get-mailboxstatistics?view=exchange-ps
OR
https://learn.microsoft.com/en-us/powershell/module/exchange/get-exomailbox?view=exchange-ps

Same with the SharePoint:
https://learn.microsoft.com/en-us/powershell/module/sharepoint-online/get-sposite?view=sharepoint-ps 

It'll need a bit more research; the other problem is that on-prem Exchange and SharePoint are, as you said, quite rare so testing could be an issue.
